spec/models/claim/geckoboard_dataset_spec.rb

```diff
@@ -21,6 +21,27 @@
     end
   end
 
+  it "correctly reports a claim's SLA status when the claim is not approaching the check deadline" do
+    ClimateControl.modify ENVIRONMENT_NAME: "test" do
+      claim_sla_ok = build(:claim, :submitted, submitted_at: (Claim::CHECK_DEADLINE - Claim::CHECK_DEADLINE_WARNING_POINT - 1.week).ago)
+      expect(Claim::GeckoboardDataset.new(claims: [claim_sla_ok]).data_for_claims([claim_sla_ok]).first[:sla_status]).to eq("ok")
```

Comment:
This is a bit tricky to follow. Might be better to have the dataset initialized as a variable first?

```ruby
dataset = Claim::GeckoboardDataset.new # Given that we don't have to pass in the claims
data = dataset.data_for_claims([claim_sla_ok])

expect(data.first[:sla_status]).to eq("ok")
```

Reply:
You could even have the blank `dataset` set up in a `let` earlier in the tests to DRY things up